readme.md

```diff
@@ -90,6 +90,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Education](https://github.com/hsavit1/Awesome-Swift-Education)
 	- [Playgrounds](https://github.com/uraimo/Awesome-Swift-Playgrounds)
 - [Python](https://github.com/vinta/awesome-python)
+	- [Asyncio](https://github.com/timofurrer/awesome-asyncio)
```

Comment:
Add a very short description about `asyncio`. See other items with description for what format to use.